plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/ProductVariationTemplate.php

```diff
@@ -455,11 +455,30 @@ private function add_shipping_group_blocks() {
 				],
 			]
 		);
+		// Virtual section.
+		$shipping_group->add_section(
+			[
+				'id'    => 'product-variation-virtual-section',
+				'order' => 20,
+			]
+		)->add_block(
+			[
+				'id'         => 'product-variation-virtual',
+				'blockName'  => 'woocommerce/product-toggle-field',
+				'order'      => 10,
+				'attributes' => [
+					'property'       => 'virtual',
+					'checkedValue'   => false,
+					'uncheckedValue' => true,
```

Comment:
Can we use one boolean instead of two here?

Reply:
Nope, because it can be use also for enums or any other type. You should take a look at [checkedvalue](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/blocks/generic/checkbox/README.md#checkedvalue) and [uncheckedvalue](https://github.com/woocommerce/woocommerce/blob/trunk/packages/js/product-editor/src/blocks/generic/checkbox/README.md#uncheckedvalue) props from the checkbox block.

Reply:
Oh, now I get it! Thank you for the clarification 🙂